addons/addon-internal-auth-dep-check/README.md

```diff
@@ -0,0 +1,7 @@
+# Addon for checking if environment is ready to upgrade after internal auth deprecation
+
+The purpose of this add-on is check that the environment is ready for deployment after upgrading to the version of Service Workbench that has deprecated internal authentication.
```

Comment:
The purpose of this add-on is _to_ check ...

Reply:
Fixed